Flepper.Tests.Integration/Flepper.Tests.Integration.csproj

```diff
@@ -6,6 +6,10 @@
     <IsPackable>false</IsPackable>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
```

Comment:
C# 7.1 only in debug?

Reply:
https://github.com/Flepper/flepper/pull/90/commits/ce217664edfba78c24b7b1351858097a8cb405b5